src/user/supla_esp_devconn.h

```diff
@@ -32,6 +32,8 @@ void DEVCONN_ICACHE_FLASH supla_esp_channel_value_changed(int channel_number, ch
 void DEVCONN_ICACHE_FLASH supla_esp_channel_extendedvalue_changed(unsigned char channel_number, TSuplaChannelExtendedValue *value);
 void DEVCONN_ICACHE_FLASH supla_esp_devconn_send_channel_values_with_delay(void);
 void DEVCONN_ICACHE_FLASH supla_esp_devconn_system_restart(void);
+void ICACHE_FLASH_ATTR supla_esp_em_get_value(
```

Comment:
Do not change the file
src / user / supla_esp_devconn.h

all you need to move to the area of ​​your own board.